pkg/demoinfocs/game_state.go

```diff
@@ -101,6 +102,11 @@ func (gs gameState) Rules() GameRules {
 	return gs.rules
 }
 
+// Hostages returns a map from entity-IDs to all hostages.
+func (gs gameState) Hostages() map[int]*common.Hostage {
```

Comment:
Would it make more sense to return a slice here?

If we really want to return a map we should make a defensive copy of it, so nobody accidentally tries to mess with it
see https://github.com/markus-wa/demoinfocs-golang/blob/master/pkg/demoinfocs/game_state.go#L270-L293

but as there's probably not too many hostages on a map and I'm not sure consumers need to access them by entity ID id recommend returning a slice.

but let me know if you disagree :slightly_smiling_face:

Reply:
I agree, I now return a slice.